TriforkSwiftExtensions/TriforkSwiftExtensions/UITextView/UITextViewExtensions.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UITextViewExtensions.swift
+//  TriforkSwiftExtensions
+//
+//  Created by Kim de Vos on 26/04/2019.
+//  Copyright © 2019 Trifork A/S. All rights reserved.
+//
+
+import UIKit
+
+extension UITextView {
+    func setZeroInset() {
```

Comment:
I think the name should be `setTextContainerInsetToZero()` instead, since the `UITextView` inherits from `UIScrollView`, so it could also refer to `contentInset`.

Reply:
🚀